src/components/event/EventWindow.tsx

```diff
@@ -33,7 +33,9 @@ function EventWindow() {
 		if (node) {
 			eventsStore.scrollToEvent(node.eventId);
 		}
-		eventsStore.selectNode(node);
+		if (!node?.isUnknown) {
```

Comment:
id rather move this check inside of `eventsStore.selectNode` instead of adding everywhere